adafruit_uc8151d.py

```diff
@@ -38,15 +38,65 @@
     b"\x50\x01\x97"  # CDI setting
 )
 
+_GRAYSCALE_START_SEQUENCE = (
+    b"\x04\x80\xc8"  # Power on
+    b"\x00\x01\xbf"  # Panel setting
+    b"\x50\x01\x97"  # CDI setting
+    # Common voltage
+    b"\x20\x2a"
+    b"\x00\x0A\x00\x00\x00\x01"
+    b"\x60\x14\x14\x00\x00\x01"
+    b"\x00\x14\x00\x00\x00\x01"
+    b"\x00\x13\x0A\x01\x00\x01"
+    b"\x00\x00\x00\x00\x00\x00"
+    b"\x00\x00\x00\x00\x00\x00"
+    b"\x00\x00\x00\x00\x00\x00"
+    # White to White
+    b"\x21\x2a"
+    b"\x40\x0A\x00\x00\x00\x01"
+    b"\x90\x14\x14\x00\x00\x01"
+    b"\x10\x14\x0A\x00\x00\x01"
+    b"\xA0\x13\x01\x00\x00\x01"
+    b"\x00\x00\x00\x00\x00\x00"
+    b"\x00\x00\x00\x00\x00\x00"
+    b"\x00\x00\x00\x00\x00\x00"
+    # Black to White
+    b"\x22\x2a"
+    b"\x40\x0A\x00\x00\x00\x01"
+    b"\x90\x14\x14\x00\x00\x01"
+    b"\x00\x14\x0A\x00\x00\x01"
+    b"\x99\x0B\x04\x04\x01\x01"
+    b"\x00\x00\x00\x00\x00\x00"
+    b"\x00\x00\x00\x00\x00\x00"
+    b"\x00\x00\x00\x00\x00\x00"
+    # White to Black
+    b"\x23\x2a"
+    b"\x40\x0A\x00\x00\x00\x01"
+    b"\x90\x14\x14\x00\x00\x01"
+    b"\x00\x14\x0A\x00\x00\x01"
+    b"\x99\x0C\x01\x03\x04\x01"
+    b"\x00\x00\x00\x00\x00\x00"
+    b"\x00\x00\x00\x00\x00\x00"
+    b"\x00\x00\x00\x00\x00\x00"
+    # Black to Black
+    b"\x24\x2a"
+    b"\x80\x0A\x00\x00\x00\x01"
+    b"\x90\x14\x14\x00\x00\x01"
+    b"\x20\x14\x0A\x00\x00\x01"
+    b"\x50\x13\x01\x00\x00\x01"
+    b"\x00\x00\x00\x00\x00\x00"
+    b"\x00\x00\x00\x00\x00\x00"
+    b"\x00\x00\x00\x00\x00\x00"
+)
+
+
 _STOP_SEQUENCE = b"\x50\x01\xf7" b"\x07\x01\xA5"  # CDI setting  # Deep Sleep
 # pylint: disable=too-few-public-methods
 class UC8151D(displayio.EPaperDisplay):
     r"""UC8151D driver
-
     :param bus: The data bus the display is on
     :param \**kwargs:
         See below
```

Comment:
I think it's getting confused by indentation like this, so it's not even your PR.  We updated the CI recently, so this is probably new.  I'm made a PR for it, and either you can add/merge in those changes and I can approve if it or you can wait for it to get merged if you'd like :)